src/components/Navigation/TableOfContents/TableOfContentsItem.tsx

```diff
@@ -72,36 +72,35 @@ const TableOfContentsItem: FC<TableOfContentsItemProps> = ({
       </Button>
       {children && (
         <AnimatePresence>
```

Comment:
AnimatePresence wrapper is not needed anymore AFAIK, since we are not using the exit animation

Reply:
removed `AnimatePresence` component and exit height from the `ChildContainer` component